pages/_app.tsx

```diff
@@ -1,17 +1,39 @@
-import "../styles/globals.css";
 import "@rainbow-me/rainbowkit/styles.css";
```

Comment:
Do we want their styles? Maybe they are just good defaults and we'll override things later?

Reply:
Yea, they are decent defaults and we can refine later.